tests/libclient/libclient.go

```diff
@@ -18,3 +18,29 @@
  */
 
 package libclient
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I think the alias fits `metav1` (like the original).

Reply:
DONE